guides/rif-relay/deployment.md

```diff
@@ -0,0 +1,280 @@
+          | multiSendAddress        | MultiSend              |
+    
+        Now we need to execute the following command, to deploy the multisig contract:
+
+          ```
+          npx truffle --network regtest exec tasks/create-safe.js --owners="0x7986b3DF570230288501EEa3D890bd66948C9B79,0x0a3aA774752ec2042c46548456c094A76C7F3a79,0xCF7CDBbB5F7BA79d3ffe74A0bBA13FC0295F6036,0x39B12C05E8503356E3a7DF0B7B33efA4c054C409"
+          ```
+          > All this accounts are the first 4 accounts from the regtest blockchain that are provided by default. 
+
+        The last step is to deploy the collector contract. To do that we need to create a configuration file with the following structure and save it as deploy-collector.input.json:
+
+        ```json
+        {
+          "collectorOwner": "0xfC6B663863722F563a9a3234E15493d2b7c365E8",
+          "partners": [
+              {
+                  "beneficiary": "0x7986b3DF570230288501EEa3D890bd66948C9B79",
+                  "share": 20
+              },
+              {
+                  "beneficiary": "0x0a3aA774752ec2042c46548456c094A76C7F3a79",
+                  "share": 35
+              },
+              {
+                  "beneficiary": "0xCF7CDBbB5F7BA79d3ffe74A0bBA13FC0295F6036",
+                  "share": 13
+              },
+              {
+                  "beneficiary": "0x39B12C05E8503356E3a7DF0B7B33efA4c054C409",
+                  "share": 32
+              }
+          ],
+          "tokenAddress": "0x726ECC75d5D51356AA4d0a5B648790cC345985ED",
+          "remainderAddress": "0xc354D97642FAa06781b76Ffb6786f72cd7746C97"
+        }
+        ```
+
+        Now we only need to execute the following command:
+
+        ```
+        npx truffle --network regtest exec tasks/deploy-collector.js --collectorConfig="deploy-collector.input.json" --outputFile="output.json"
+        ```
+
+        The collector is ready and can start receiving fees. 
+    - Testnet
+      * TBD
+    - Mainnet
+      * TBD
+
+- 3) Allow Tokens
+  * RIF Relay only accepts whitelisted tokens. There are multiple reasons to just allow whitelisted tokens but the main one is to ensure that RIF Relay is just accepting tokens that represent value for the sponsor.
+
+  [](#top "collapsible")
+    - Regtest
+      * In the RIF Relay Contracts we will need to execute the command:
+        ```
+        npm run allowTokens 0x726ECC75d5D51356AA4d0a5B648790cC345985ED regtest
+        ```
+        > The allowTokens script assumes that you are in Regtest and uses the account[0] as the owner of the contracts and that’s important because only the owner can allow tokens.
+    - Testnet
+      * To whitelist a token on Testnet we need to execute the `acceptToken(address token)` on the Relay Verifiers:
+
+        * `SmartWalletDeployVerifier`
+        * `SmartWalletRelayVerifier`
+
+        > The contract execution needs to be done by the owner that is the account that did the deployment.
```

Comment:
Although this is true, why didn't we include the command `npm run allowTokens <token> testnet`?

Reply:
The allow tokens with testnet, will use the account 0, isn't it?

Reply:
Yes, it will. And what's the issue with it?